src/util.go

```diff
@@ -106,21 +106,25 @@ func setDefaultValues(config *Config) {
 	// Set default values for string and boolean fields if they are nil (not present in the YAML)
 	if config.Directive == nil {
 		defaultDirectiveValue := "rhc-worker-script"
+		log.Infof("config 'directive' value is empty default value (%s) will be used", defaultDirectiveValue)
 		config.Directive = &defaultDirectiveValue
 	}
 
 	if config.VerifyYAML == nil {
 		defaultVerifyYamlValue := true
+		log.Infof("config 'verify_yaml' value is empty default value (%v) will be used", defaultVerifyYamlValue)
```

Comment:
What about using %t instead of %v? Taken from the golang docs: 

```
%t	the word true or false
```

```suggestion
		log.Infof("config 'verify_yaml' value is empty default value (%t) will be used", defaultVerifyYamlValue)
```

Reply:
changed